app/views/spree/admin/subscriptions/edit.html.erb

```diff
@@ -1,7 +1,7 @@
 <% content_for(:page_title) { t('.title') } %>
 
 <% content_for :page_actions do %>
-  <li><%= button_link_to t('.back'), admin_subscriptions_path %></li>
+  <li><%= button_to t('.back'), admin_subscriptions_path %></li>
```

Comment:
should this line be `<li><%= button_to I18n.t('.back'), admin_subscriptions_path %></li>` ?

i see the older version didn't have that, but not sure if there's a shorthand method that works with just `t`

Reply:
From https://api.rubyonrails.org/v5.2.3/classes/ActionView/Helpers/TranslationHelper.html it looks like `t` resolves to `I18n.t`.